src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  font-family: Avenir, serif;
+}
+
+.search-box {
+  position: relative;
+  width: 100%;
+}
+
+.search-box-big {
+  margin: 20px 0;
+}
+
+.search-box input[type="search"] {
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  outline: none;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: top 25px left 26px;
+  background-size: 19px 19px;
+}
+
+.search-box-big input[type="search"] {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+}
```

Comment:
move it tot search-box
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.